test/ts/to-file.es5.cjs

```diff
@@ -0,0 +1,58 @@
+'use strict'
```

Comment:
Can you ignore this file?

Reply:
This file is still here.

Reply:
yep, was working on it. The thumb was a confirmation that I would do it :) . Done now.